### PR TITLE
feat: update the colors and rotation

### PR DESCRIPTION
**Pull request by sebinbenjamin**
This is another fun little PR to demo how PRs work.

**What changed**
- Updated the background colour to green as discussed.
- Reversed the logo animation.

**Screenshots**
image

### CODE REVIEW

**Review by J-eld — CHANGES_REQUESTED** [review]
Change it ALL!

**Comment by sebinbenjamin** [comment]
> Change it ALL!
 
 I've changed it to red as requested.

image

File: src/App.css [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
 @media (prefers-reduced-motion: no-preference) {
   .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+    animation: App-logo-spin infinite 20s alternate-reverse;
   }
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color: red;
```
Comment (J-eld) [inline src/App.css]: don't like it, change to black

Comment (sebinbenjamin) [reply src/App.css]: I've changed it to look like this

image

**Comment by J-eld** [comment]
Ship it! 🚢

**Review by J-eld — APPROVED** [review]

**Pull request merged**